test/test.js

```diff
@@ -25,6 +28,25 @@ describe('Walder', function () {
       expect(() => new Walder()).to.throw('Configuration file is required.')
     });
 
+    async function activateFromConfigFile(configFileName) {
+      const configFile = path.resolve(__dirname, configFileName);
+      const walder = new Walder(configFile);
+      try {
+        await walder.activate();
+        throw 'not an instance of Error'; // on request: not using chai-as-promised here, using nice solution from https://arpadt.com/articles/testing-asynchronous-code-with-mocha
```

Comment:
Can the error be more descriptive? Like it's not an instance of an error, sure, but what about it?

Reply:
Clarified that.